packages/@glimmer/syntax/test/loc-node-test.ts

```diff
@@ -134,29 +138,6 @@ test('html elements', () => {
   }
 });
 
-test('html elements with paths', () => {
-  let ast = parse(`
-    <Foo as |bar|>
-      <bar.x.y class='bar'/>
```

Comment:
I didn't see a replacement test for this scenario -- why was the test removed? (or did I miss an equiv test?)

Reply:
Because I only pushed the part one of two (the block statement part) not the element one so far